addons/mail/static/tests/helpers/mock_models.js

```diff
@@ -124,7 +124,7 @@ class MockModels {
                     attachment_ids: { string: "Attachments", type: 'many2many', relation: 'ir.attachment', default: [] },
                     author_id: { string: "Author", type: 'many2one', relation: 'res.partner', default() { return this.currentPartnerId; } },
                     body: { string: "Contents", type: 'html', default: "<p></p>" },
-                    date: { string: "Date", type: 'datetime' },
+                    date: { string: "Date", type: 'datetime', default() { return moment.utc().format("YYYY-MM-DD HH:mm:ss"); } },
```

Comment:
Also just `moment()`, I don't think we need `utc()`

Reply:
We need UTC, because the database use UTC. Otherwise it add 2 hours.

Reply:
ok for this one